lib/winston-logstash-udp.js

```diff
@@ -46,7 +46,13 @@ util.inherits(LogstashUDP, winston.Transport);
 winston.transports.LogstashUDP = LogstashUDP;
 
 LogstashUDP.prototype.connect = function() {
+    var self = this;
     this.client = dgram.createSocket('udp4');
+
+    this.client.on('error', function(err) {
+        // Skipping DNS errors
+        self.emit('error', err);
```

Comment:
Makes sense.  Can you modify the comment to reflect this?  Then we just need a test to check that the error event is emitted and I'll be happy to merge this.